content/pt-br/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -0,0 +1,361 @@
+
+Ao escalonar para baixo, o Replicaset controller escolhe quais pods irá deletar ordenando os pods disponíveis para priorizar quais pods seram escalonados para baixo seguindo o seguinte algoritmo geral:
+ 1. Pods pendentes (e não agendáveis) são decaídos primeiro
+ 2. Se a anotação `controller.kubernetes.io/pod-deletion-cost` estiver definida, então o pod com o menor valor será priorizado primeiro.
+ 3. Pods em nós com mais réplicas são decaídos primeiro que pods em nodes com menos réplicas.
+ 4. Se a data de criação dos pods for diferente, o pod que foi criado mais recentemente vem antes que o pod mais antigo (as datas de criação são guardados em uma escala logarítmica caso o [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) `LogarithmicScaleDown` esteja habilitado)
+
+Se o Pod obedecer todos os items acima simultaneamente, a seleção é aleatória.
+
+### Custo de deleção de Pods
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
+
+Utilizando a anotação [`controller.kubernetes.io/pod-deletion-cost`](/docs/reference/labels-annotations-taints/#pod-deletion-cost),
+usuários podem definir uma preferência em relação à quais pods serão removidos primeiro caso o ReplicaSet precise escalar para baixo.
+
+A anotação deve ser definida no pod, com uma variação de [-2147483647, 2147483647]. Isso representa o custo de deletar um pod comparado com outros pods que pertencem à esse mesmo ReplicaSet. Pods com um custo de deleção menor são eleitos para deleção antes de pods com um custo maior.
+
+O valor implícito para essa anotação para pods que não a tem definida é 0;  valores negativos são permitidos.
+Valores inválidos serão rejeitados pelo servidor API.
+
+Esse recurso está em beta e é habilitado por padrão. Você consegue desabilita-lo usando o 
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`PodDeletionCost` ambos no kube-apiserver e no kube-controller-manager.
+
+{{< note >}}
+- Esse recurso é honrado baseado no melhor esforço, portanto não oferece qualquer garantia na ordem de deleção dos pods.
+- Usuários são recomendados à evitar atualizações frequentes em anotações, como gerar atualizações baseando-se em alguma métrica, porque fazendo isso irá criar um número significante de atualizações de pod para o apiserver.
+{{< /note >}}
+
+#### Exemplo de caso de uso
+Os diferentes pods de uma aplicação podem ter níves de utilização divergentes. Ao escalar para baixo, a aplicação pode preferir remover os pods com a menor utilização. Para evitar atualizações frequentes nos pods, a aplicação deve atualizar `controller.kubernetes.io/pod-deletion-cost` uma vez antes de expedir o decaimento das réplicas (configurando a anotação para um valor proporcional ao nível de utilização do pod). Isso funciona se a própria aplicação controlar a escala para baixo; por exemplo, o pod condutor de um deployment de Spark.
```

Comment:
Some things in this paragraph:
* There's a typo in "níves" (I think you meant "níveis").
* "decaimento" sounds strange, is there another term you could use here?
* "controlar a escala para baixo" is also sounding strange. Is there another way you could phrase this to give the actual meaning of reducing the number of Pods?
* Keep the Kubernetes terms in title case (i.e. "Pod" in lieu of "pod", "Deployment" in lieu of "deployment").